src/components/Modal/index.js

```diff
@@ -30,25 +30,24 @@ export default function Modal({
 
   const [isRoundModal, setIsRoundModal] = useState(isInterim);
 
-  const roundResult = () => {
-    setIsRoundModal(false);
-  };
-
   return (
     <>
       {!isfinishResult ? (
-        <ModalDiv onClick={handleClick}>
-          <p>{isRoundModal ? "중간 결과" : "라운드 결과"}</p>
-          {isRoundModal ? (
-            <InterimFindings scoreData={scoreBoard} selectData={selectBoard} round={round} />
-          ) : (
-            <WrapResult>
-              {roundResultData().map((it) => (
-                <TeamResult key={`modal_${it.team}`} team={it.team} cardXY={it.cardXY} point={it.point} />
-              ))}
-            </WrapResult>
-          )}
-        </ModalDiv>
+        <>
+          <ModalDiv onClick={handleClick}>
+            <p>{isRoundModal ? "중간 결과" : "라운드 결과"}</p>
+            {isRoundModal ? (
+              <InterimFindings scoreData={scoreBoard} selectData={selectBoard} round={round} />
+            ) : (
+              <WrapResult>
+                {roundResultData().map((it) => (
+                  <TeamResult key={`modal_${it.team}`} team={it.team} cardXY={it.cardXY} point={it.point} />
```

Comment:
이 props네이밍은 data 혹은 roundInfo 정도가 괜찮아 보입니다.
it이라고하면 그 it의 데이터가 뭐가 들은건지 정확하게 명시적이지 않아보여서 추천드립니다.
```suggestion
                {roundResultData().map((data) => (
                  <TeamResult key={`modal_${data.team}`} team={data.team} cardXY={data.cardXY} point={data.point} />
```